include/UI/UIView.h

```diff
@@ -220,16 +255,45 @@ class UIView
 
   vec2f m_histogram_point_1;
   vec2f m_histogram_point_2;
-  float m_histogram_quantiles[2]; 
+  float m_histogram_quantiles[2];
+
 
-  
   void load_ocean_color_maps();
 
   std::vector<std::string> m_ocean_color_maps_names;
 
   std::string m_color_map_directory;
```

Comment:
why prepend all the variable names with `m_`?

Reply:
we use the convention of name member variables are prepended with m_